baseapp/baseapp_test.go

```diff
@@ -79,7 +79,7 @@ func TestInitChainer(t *testing.T) {
 
 	// initChainer sets a value in the store
 	var initChainer sdk.InitChainer = func(ctx sdk.Context, req abci.RequestInitChain) abci.ResponseInitChain {
-		store := ctx.KVStore(capKey)
```

Comment:
The version before your PR is more go-standard.
`GetXYZ` is java-style.

https://golang.org/doc/effective_go.html#Getters

Reply:
Agree with Frey.